src/pages/home/report/ReportActionsView.js

```diff
@@ -37,6 +36,9 @@ import PopoverReportActionContextMenu from './ContextMenu/PopoverReportActionCon
 import variables from '../../../styles/variables';
 import MarkerBadge from './MarkerBadge';
 import Performance from '../../../libs/Performance';
+import EmptyStateAvatars from '../../../components/EmptyStateAvatars';
+import {isDefaultRoom} from '../../../libs/reportUtils';
```

Comment:
```suggestion
import * as ReportUtils from '../../../libs/reportUtils';
```

Reply:
i don't think so that it is efficient way but updated